reference/redox.v1.yaml

```diff
@@ -0,0 +1,35 @@
+openapi: 3.0.0
+x-stoplight:
+  id: gpm2ktdetzht7
+info:
+  title: Redox
+  version: '1.0'
+  description: |-
+    This spec is used for generating models for incomming Redox messages. In the future it may also be used for generating an SDK library for pushing results to Redox.
+
+    To enable code generation for a specific model you should reference it in the request body `/endpoint` API endpoint.
+servers:
+  - url: 'https://api.redoxengine.com/endpoint'
+paths:
+  /endpoint:
+    post:
+      summary: Send Data
+      tags: []
+      responses: {}
+      operationId: SendData
+      requestBody:
+        content:
+          application/json:
+            schema:
+              anyOf:
+                - $ref: '#/components/schemas/NewOrder'
```

Comment:
This is a bit confusing. We won't be sending a `NewOrder` to Redox, right? We'll be receiving one, and sending one of (`NewResults` | `NewMedia` | `NewNotes`) response back.

But I guess this is just to drive a codegen tool to produce code, it doesn't really matter.

Reply:
This was the only way to get model generation working. Initially it was no-op endpoint then I saw that Redox exposes a single endpoint for all data models. We could just add what we need here, or even add all models. I didn't try generating a client.